src/main.js

```diff
@@ -379,8 +385,20 @@ let load = loader({
   },
 
   all: {
-    requires: ['provisioner', 'server'],
-    setup: async ({provisioner, server}) => {
+    requires: ['cfg', 'provisioner', 'server', 'queue', 'WorkerType'],
+    setup: async ({cfg, provisioner, server, queue, WorkerType}) => {
+      if (cfg.app.publishQueueMetadata) {
+        await queue.declareProvisioner(cfg.app.provisionerId, {
+          stability: cfg.app.stability,
+          expires: taskcluster.fromNow('36 hours'),
+          description: cfg.app.description,
+        });
+
+        await WorkerType.scan({}, {
+          handler: wt => wt.declareWorkerType(),
```

Comment:
If this really is something which is wanted and it's supposed to be done at an interval, I'd rather express that by putting it in a taskcluster-lib-iterate and have it run periodically instead of relying exclusively on the 24-hour reboot.

Reply:
This follows the model of docs, schemas, and references.

Does lib-iterate guarantee an immediate call, or does it wait `waitTime` ms first?

Reply:
It does :) https://github.com/taskcluster/taskcluster-lib-iterate/pull/5